src/components/victory-bar.jsx

```diff
@@ -101,6 +109,16 @@ export default class VictoryBar extends React.Component {
      */
     categoryLabels: React.PropTypes.array,
     /**
+     * The colorScale prop is an optional prop that defines the color scale the chart's bars
+     * will be created on. This prop should be given as a string, which will one of the five
+     * baked-in color scales: "victory", "grayscale", "red", "bluePurple", and "yellowBlue".
+     * If it is not defined, the default Victory grayscale will be used, and if the fill
+     * property on the dataAttributes prop is defined, it will overwrite the colorScale prop.
+     */
+    colorScale: React.PropTypes.oneOf([
```

Comment:
it seems like a nice to have. array rather than an array of objects... and you get repeating colors without having to explicitly define them.